tests/test_config.py

```
@@ -1,6 +1,7 @@
 """Tests for `config.py`"""
 
-from omegaconf import OmegaConf
+from omegaconf import OmegaConf, open_dict
+from copy import deepcopy
```

Ruff

`copy.deepcopy` imported but unused

Remove unused import: `copy.deepcopy`

(F401)